nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,347 @@
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Future Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:retainUntil"
+            group-limit="10"
+            data="{{futureRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Expired Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="record:retainUntil"
+            group-limit="10"
+            data="{{expiredRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card class="retentionStatus" heading="[[i18n('retentionAnalytics.retentionStatus')]]">
+            <chart-pie
+              values="[[_valuesRetentionStatus(noRetention, futureRetention, expiredRetention)]]"
+              labels="[[_labelsRetentionStatus()]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+
+          <!-- LegalHold by Document Type -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:primaryType"
+            where='{"ecm:hasLegalHold": "true"}'
+            group-limit="20"
+            data="{{legalHoldByDocType}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.legalHoldByDocType')]]">
+            <chart-pie
+              values="[[_values(legalHoldByDocType)]]"
+              labels="[[_labels(legalHoldByDocType)]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+        </div>
+
+        <!-- Expiration next 12 months -->
+        <nuxeo-repository-data
+          start-date="[[_formatDateNow()]]"
+          end-date="[[_formatDateFutureInMonths(12)]]"
+          with-date-intervals="month"
+          date-field="ecm:retainUntil"
+          data="{{expirationPerMonth}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <nuxeo-card class="chartBar" heading="[[i18n('retentionAnalytics.expirationNext12Months.heading')]]">
+          <chart-bar
+            labels="[[_futureExpirationLabels(expirationPerMonth)]]"
+            values="[[_values(expirationPerMonth)]]"
+            options='{ "legend": { "display": false }, "animation": false }'
```

Comment:
The initial things that was done, added some code to show the labels of each pie. Without to have the tooltip. As we spoke, we can ask for UI team feedbacks / insights. But it seems that https://github.com/emn178/chartjs-plugin-labels is does it very well, there is a great demo https://emn178.github.io/chartjs-plugin-labels/samples/demo/

Reply:
I am following the other analytics examples on webui. Yes I'll take a look at the demo, but I think this change should be applied more generically to the element -  @nuxeo/chart-elements/chart-bar.js

Reply:
I also discussed it with Julien and he agrees we should add it as a future feature, and apply it more generically to all nuxeo charts.

Reply:
I think you have a ticket for this purpose. you can added here for tracking.